xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/colorThemeMappingOutput.vm

```diff
@@ -35,4 +35,7 @@
 #mapping("notificationSuccessColor", "@brand-success")
 #mapping("notificationInfoColor", "@brand-info")
 #mapping("notificationWarningColor", "@brand-warning")
-#mapping("notificationErrorColor", "@brand-danger")
\ No newline at end of file
+#mapping("notificationErrorColor", "@brand-danger")
+#mapping("notificationEmailBg", "@notification-email-bg")
+#mapping("notificationEmailInnerBg", "@notification-email-inner-bg")
+#mapping("notificationEmailTextColor", "@notification-email-text-color")
```

Comment:
Also if we were to add new variables, not sure if we should also add the mapping. The mappings is supposed to be used for the old CT variables, not for the new ones. @gdelhumeau wdyt?

Reply:
Indeed, it does not seem necessary here.